apps/codeforafrica/src/components/GuidingPrinciplesCard/GuidingPrinciplesCard.js

```diff
@@ -0,0 +1,36 @@
+import { RichTypography } from "@commons-ui/core";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import React from "react";
+
+const GuidingPrinciplesCard = React.forwardRef(function GuidingPrinciplesCard(
+  props,
+  ref
+) {
+  const { title, description, icon } = props;
+  return (
+    <Card
+      sx={{
+        borderRadius: 0,
+        boxShadow: "none",
+        marginTop: { lg: "40px", xs: "20px" },
+        padding: "24px",
+        width: { sm: "354px", md: "354px", lg: "324px", xl: "354px" },
```

Comment:
`sm`, `lg`, `xl` are all 354; `md` is 329.33. Another approach is to use Grid and specify number of columns.